### ISSUE TITLE
Dwc-A fieldsEnclosedBy default value

### ISSUE THREAD

**Issue by cgendreau** [tdwg/dwc#140]
Just to be sure, the default value in the Darwin Core Archive `meta.xml` for the attribute `fieldsEnclosedBy` should be `"`.

In the [Darwin Core Text Guide](http://tdwg.github.io/dwc/terms/guides/text/#coreTag) we can read:
> Specifies the character used to enclose (mark the start and end of) each field. CSV files frequently use the double quote character ("), but the default is no enclosing character. Note that a comma separated value file that has commas within the content of any field must have an enclosing character.

But the Default column says `"`. The [XSD](https://github.com/tdwg/dwc/blob/master/text/tdwg_dwc_text.xsd#L32) also points to  `"`.

It seems the issue is in the Description.

**Comment by ansell** [comment]
Following the CSV profile defined in RFC4180 Section 2 consistently for the defaults could be useful to encourage people to follow that as a best practice, while supporting other forms using metadata.

https://tools.ietf.org/html/rfc4180#section-2

**Comment by timrobertson100** [comment]
I can confirm this was an oversight in the description text only.  

The intention was to assume `"` encoding which is what most tools did at the time I wrote it unless explicitly stated otherwise in a `meta.xml`.  Thus an `occurrence.csv` file like this would be supported without any `meta.xml`.

```
occurrenceID,occurrenceRemarks
1,Looking good
2,"looking good, mate"
```

**Comment by tucotuco** [comment]
Fixed in 03bc7e182fbce323ae66caedef6da3a1451ca6b8
